pkg/controller/volume/attach_detach_controller.go

```diff
@@ -309,10 +299,11 @@ func (adc *attachDetachController) processPodVolumes(
 
 		} else {
 			// Remove volume from desired state of world
-			uniqueVolumeName, err := attachableVolumePlugin.GetUniqueVolumeName(volumeSpec)
+			uniqueVolumeName, err := attachdetach.GetUniqueDeviceNameFromSpec(
+				attachableVolumePlugin, volumeSpec)
 			if err != nil {
 				glog.V(10).Infof(
-					"Failed to delete volume %q for pod %q/%q from desiredStateOfWorld. GetUniqueVolumeName failed with %v",
+					"Failed to delete volume %q for pod %q/%q from desiredStateOfWorld. GenerateUniqueDeviceName failed with %v",
```

Comment:
Nit: log messages are supposed to start with lowercase letters -- no need to fix now, perhaps in a follow up.